config/v1/types_infrastructure.go

```diff
@@ -1568,7 +1568,7 @@ type PowerVSServiceEndpoint struct {
 	// Power Cloud - https://cloud.ibm.com/apidocs/power-cloud
 	//
 	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:Pattern=`^[a-z0-9-]+$`
+	// +kubebuilder:validation:Pattern=`^[A-z0-9-]+$`
```

Comment:
Is this a valid regex?

Reply:
Should be yep. Wrote a small test here.

https://play.golang.com/p/jZKKt0eevZ0

Reply:
Perhaps you can be safer and use [a-zA-Z0-9-]
?

Reply:
I'm fine with that as well if folks think it's better, but the proposed regex should be expressing the same range. I also tested it against alllowercase for backwards compatability

https://play.golang.com/p/LR54k52gDdX

Reply:
@hamzy pointed out to me that `A-z` includes `[ \ ] ^ _ \` which we do not want to be valid! resubmitting with his suggestion